apps/modernization-ui/src/apps/patient/data/phoneEmail/PhoneEmailEntryFields.tsx

```diff
@@ -87,8 +87,8 @@ export const PhoneEmailEntryFields = ({ orientation = 'horizontal' }: PhoneEmail
                 name="countryCode"
                 rules={{
                     pattern: {
-                        value: /^\+?\d{1,3}$/,
-                        message: 'A Country code should be 1 to 3 digits'
+                        value: /^\+?\d{1,20}$/,
+                        message: 'A country code should be 1 to 20 digits'
```

Comment:
issue:  I think "Country" was capitalized so that message casing matched the field label.